### PR TITLE
Feature/intl

### PR DESCRIPTION
**Pull request by maximblack**
Based mostly on @enten reaction repository. [https://github.com/enten/reduction](url)

# Features:
**intl:extractLocaleData** - extracts locale data from "react-intl" and "intl" packages into "src/locale-data" directory;
**intl:extractMessages** - extracts all messages used in project components, the filename wildcard excludes the ".test"
**Using import() from Webpack 2.** - for dynamic imports of locale data
**Stores all messages in Redux and dynamically adding on locale change** - without duplication. using Axios now for requests
**Using safeConfigGet** - for locales properties on client side

# Features plans:
- Auto extract messages on file change
- Apollo integration (for fetching locale messages)
- Gui editor for translation messages?
- Auto extract locales data on config change?

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Awesome thanks @maximblack!!!  I really appreciate you taking on this feature. :)

I am about to do another release.  I am renaming the current `feature/redux` branch to `feature/redux-opinionated`, and then I hope to have a "simple" redux branch that doesn't include flow and the custom task routes implementation.  I will give you a heads up when this rolls forward and we can work together to get this PR updated appropriately.

@enten - I would appreciate your eyes on this one.

**Comment by eurobob** [comment]
How would I go about merging this PR into my app? i tried `git merge upstream/feature/intl` but it didn't work

File: config/index.js [hunk 1]
```diff
@@ -118,7 +124,7 @@ const config = {
   // and it is served by the "server" bundle.
   serviceWorker: {
     // Enabled?
-    enabled: true,
+    enabled: false,
```
Comment (kikoanis) [inline config/index.js]: @maximblack is there a reason why are you disabling service worker here?

Comment (maximblack) [inline config/index.js]: There is no reason for that. Just forgot to enable it.

**Comment by maximblack** [comment]
Created pull request on "feature/redux-opinionated" [https://github.com/ctrlplusb/react-universally/pull/338](url).

**Pull request closed without merging**